### PR TITLE
api parallelization + sgp4

### PR DESCRIPTION
**Pull request by syko240**
added:
- parallezation of api calls for fetching TLE data
- sgp4 binary
- store sgp4 result to db

note, removed ignore for bin/

### CODE REVIEW

**Review by mohammed-shakir — CHANGES_REQUESTED** [review]
There is duplicate code in the Dockerfile, i left it as a comment so you can see where it is.

I think everything else looks good, although ivan might cook you for using try catch.

File: Dockerfile [hunk 1]
```diff
@@ -27,12 +30,24 @@ RUN apt-get update && apt-get install -y \
     libcurl4 \
     libssl3  \
     libjsoncpp25 \
+    python3 \
+    python3-dev \
+    python3-pip \
     libpqxx-dev
```
[the remark this answers is not in the document]
Comment (mohammed-shakir) [reply Dockerfile:36]: (including the libpqxx-dev lib)

[the remark this answers is not in the document]
Comment (syko240) [reply Dockerfile:36]: that is a very good question, didn't even notice. you have to @ suad on this. but i will fix

[the remark this answers is not in the document]
Comment (syko240) [reply Dockerfile:36]: no, there is nothing wrong, they should be there. same as in current main

[the remark this answers is not in the document]
Comment (mohammed-shakir) [reply Dockerfile:36]: chat we cooked chat?

Ait bet, LGTM!

**Review by mohammed-shakir — APPROVED** [review]
LGTM!

**Pull request merged**